lib/child-writer/writer-proxy.js

```diff
@@ -27,6 +27,15 @@ const EXIT_CODES = require('../shared/exit-codes')
 const robot = require('../shared/robot')
 const permissions = require('../shared/permissions')
 const packageJSON = require('../../package.json')
+const units = require('../shared/units')
+
+/**
+ * @summary Frequency of process heartbeat signal in milliseconds
+ * @constant
+ * @private
+ * @type {Number}
+ */
+const HEARTBEAT_TIME_MS = units.MILLISECONDS_IN_A_SECOND
```

Comment:
I guess this'll need fixing @Shou

Reply:
Thanks, and we might as well increase it considering the new timeout is 60s.